src/docs/03-Code Guidelines/06-Changelog.md

```diff
@@ -1,16 +1,5 @@
 <div class="frctl-example">
     <div class="changelog">
-        {{#each releases}}
-        <article class="changelog__item">
-            <figure class="changelog__avatar">
-                <img width="60" height="60" src="{{this.author.avatar_url}}" alt="Authored by {{this.author.login}}" />
-            </figure>
-            <div class="changelog__details">
-                <h2 class="changelog__title"><a href="{{this.html_url}}">{{this.name}}</a> ({{this.tag_name}})</h2>
-                <p class="changelog__desc">{{this.body}}</p>
-                <span class="changelog__released">{{this.published_at}}</span>
-            </div>
-        </article>
-        {{/each}}
+        <p>See a <a href="https://github.com/youseedk/dna/releases">complete list of releases</a> on GitHub.</p>

```

Comment:
Nice! Maybe add `target="_blank"` here

Reply:
I personally hate it when websites make that decision for me — I prefer not to have it.

Reply:
Ok fine with me